mxnet/inference/docker/1.7.0/py3/Dockerfile.eia

```diff
@@ -5,9 +5,10 @@ LABEL maintainer="Amazon AI"
 LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
 LABEL dlc_major_version="1"
```

Comment:
The `dlc_major_version` should be incremented to `2` as we are upgrading the python version.